mkdocs.yml

```diff
@@ -60,4 +60,9 @@ plugins:
   - search
   - social:
       cards_layout_options:
-        background_color: '#295DAA'
\ No newline at end of file
+        background_color: '#295DAA'
+  - htmlproofer:
+      enabled: !ENV [ENABLED_HTMLPROOFER, True]
+      raise_error_after_finish: True
+      raise_error_excludes:
```

Comment:
Why not simply fix this?

Reply:
I don't think it makes sense to create a heading in `types.md` for one-off types.

Let me see what I can do.

Reply:
<img width="692" alt="image" src="https://github.com/maplibre/maplibre-style-spec/assets/649392/ec24fccc-529e-435e-92ad-73d5cdfc2f39">

This is what it looks like now.